### ISSUE TITLE
Make Global SB more findable as a set of datasets

### ISSUE THREAD

**Issue by ocrasborn** [Signbank/Global-signbank#519]
* Occasion:
https://www.searchenginejournal.com/google-introduces-new-search-engine-for-finding-datasets/268337/
https://toolbox.google.com/datasetsearch
https://developers.google.com/search/docs/data-types/dataset
* How are we generally in terms of SEO, not just for the site but also for the individual datasets? Are the descriptions we currently have for datasets sensible? Can we add unique identifiers of a particular type (DOI? ISLRN?) to make them findable and citable?

**Comment by Woseseltops** [comment]
About Google dataset search: I looked at a number of datasets that are available, and noticed they are all zip or csv files you can download. To link Signbank to this search engine, I think there should be an (automatically generated) publicly available page for each dataset where you can download a recent csv export. Perhaps this csv export only contains the glosses that were marked 'in web dictionary', and the publicly available page mentions how many signs there are and what percentage is publicly available?

**Comment by susanodd** [comment]
Don't you just want to know that Datasets are available in Signbank? Not actually be able to just grab them? (i.e., without bothering to make an account)

**Comment by ocrasborn** [comment]
@Woseseltops @susanodd : Indeed we don't want people to be able download the datasets from Google, we want people to find them, so to have them end up at the dataset metadata page (from which they can request access et cetera). Once people do have access, they can (sometimes, depending on permissions) export a CSV themselves.

**Comment by Woseseltops** [comment]
>  we don't want people to be able download the datasets from Google

_Google dataset search_ only links to external webpages that contain metadata about the dataset, as well as dataset downloads. So this is not about actually handing over your dataset to Google.

> Once people do have access, they can (sometimes, depending on permissions) export a CSV themselves.

As I understand it (the FAQ is not superclear about this), to be indexed by _Google dataset search_ you have to actually provide a download on the dataset metadata page, not behind authentication, and Google has automated tools to check if there indeed is something to download.

In other words, the question is: do we want to create a CSV export button per dataset, for signs that are publicly available (in the web dictionary), that you can use without logging in? If yes, we can join _Google dataset search_ . 

My personal opinion is that we should definitely do that, as it greatly increase the chance that other researchers will build on our work - even if we currently have only 44 signs publicly available... but it's not up to me to decide :)

**Comment by ocrasborn** [comment]
Oh I see. I think I agree with you, @Woseseltops , if it is public anyway, then why not make it more findable and usable. However, let's discuss what should be in such a CSV export. Only very few fields (gloss, translation equivalent; plus the video) are currently presented to anonymous users. Perhaps it would make sense to extend this a bit, also in the public interface. I'll assign this issue for me, so I will be reminded to give it some more thought.

**Comment by Woseseltops** [comment]
I looked into this a bit and discovered that the dataset pages are already there, and are visible without logging. That means the only things that need to be done are:
* Add the total number and the number of public signs to this page.
* Add a download button (what this should do exactly is what @ocrasborn should think about)
* Make sure the page matches what is described here: https://developers.google.com/search/docs/data-types/dataset which can be verified here https://search.google.com/structured-data/testing-tool/u/0/

I already did the first one, but the code I added slows the page down. Will think of something smarter.

**Comment by ocrasborn** [comment]
@Woseseltops : In terms of the first bullet: this need not be a live count, right? Can well be generated and stored overnight. (The current counts are wrong, but you probably know that, and the total number is for to the whole database count, not the individual dataset.)
I will look into the other two issues.
